### PR TITLE
Migrating to React

### PR DESCRIPTION
**Pull request by NtshVrm**
**Note: Functionality hasn't been added yet**
- Ported all pages from HTML to JSX.
- Broken down into necessary components.
-Added Routing for all pages.

### HOMEPAGE
image

### PRODUCT LISTING
image

### WISHLIST
image

### CART
image

### AUTH PAGES
image

image

### CODE REVIEW

File: src/assets/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import mclaren from "./mclaren.png";
+import mercedes from "./mercedes.png";
+import perez from "./perez.png";
+import redbull_banner from "./redbull-banner.jpg";
+import redbull from "./redbull.png";
+import redbull_logo from "./redbull-logo.png";
+import shirt from "./shirt.png";
+import sign_in from "./sign-in.jpg";
+import sign_up from "./sign-up.jpg";
+import williams from "./williams.png";
```
Comment (HarshithaKonduru214) [inline src/assets/index.js:25]: Can directly export all the images instead of importing and then exporting.

**Review by Ash-D23 — APPROVED** [review]
Avoid migrating so many pages at once, Convert Page by Page and divide into components raise PR for that page
Structure looks good to merge

File: src/components/home/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import "./Home.css";
+
+export function Navbar() {
+  return (
+    <header className="navbar">
+      <div className="nav-brand">
+        <img
+          className="nav-logo hidden"
+          src="https://picsum.photos/1000/400"
+        ></img>
```
Comment (Pratik1005) [inline src/components/home/Navbar.jsx:18]: Just remove the </img> tag.

**Review by HarshithaKonduru214 — APPROVED** [review]

**Comment by HarshithaKonduru214** [comment]
Things I appreciate:
1. Clean file structure.
2. Broken down to good small components.

**Review by Pratik1005 — APPROVED** [review]
Other than suggested change good to go

**Pull request merged**